libs/email/src/components/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './getInTouch'
```

Comment:
**blocker:** I think we shouldn't use barrel file for exporting from `components`, we should directly export from main level `index.ts`.